src/data/faq_de.json

```diff
@@ -785,6 +785,64 @@
+                        ]
+                    },
+                    {
+                        "title": "Ich habe die Meldung bekommen, dass ich eine niedrige Risikobegegnung hatte. Am nächsten Tag war sie wieder weg. Sollte das nicht erst nach 14 Tagen gelöscht werden?",
+                        "anchor": "encounter_deletion",
+                        "active": true,
+                        "textblock": [
+                            "Sobald die App die aktuellen Diagnoseschlüssel (Positivkennungen) vom Server lädt, wird eine Überprüfung des Kontaktprotokolls durchgeführt. Dabei wird für jeden der letzten 14 Tage, für den Diagnoseschlüssel vorhanden sind, eine separate Überprüfung des Kontaktprotokolls durchgeführt. Das bedeutet also, die App gleicht die Begegnungen der letzten 14 Tage mit den Positivkennungen ab, um zu überprüfen, ob es zu einem Kontakt mit einer positiv getesteten Person gekommen ist. Ist die Meldung über eine Begegnung mit niedrigem Risiko nach einem Tag verschwunden, bedeutet das, dass die entsprechende Begegnung in diesem Fall länger als 14 Tage zurückliegt."
+                        ]
+                    },
+                    {
+                        "title": "Ich habe in der App die Meldung bekommen, dass ich zwei Begegnungen mit niedrigem Risiko hatte. Die Statusanzeige ist aber grün geblieben. Muss ich mir Sorgen machen?",
+                        "anchor": "low_risk_green",
+                        "active": true,
+                        "textblock": [
+                            "Nein, denn auf Grundlage der Daten der Risiko-Ermittlung wird das Infektionsrisiko als niedrig eingestuft, da sich die Begegnung auf einen kurzen Zeitraum und/oder einen größeren Abstand beschränkt hat. Sie müssen sich keine Sorgen machen und es besteht kein besonderer Handlungsbedarf. Es wird empfohlen, sich an die allgemein geltenden Abstands- und Hygieneregeln zu halten. Sie müssen sich bei Begegnungen mit niedrigem Risiko nicht testen lassen. Die Verhaltensvorschläge sowie Infos zum Ermittlungszeitraum als auch zum Infektionsrisiko finden Nutzer*innen in der App selbst, wenn sie auf „Niedriges Risik“ mit dem Finger tippen."
```

Comment:
Typo:
Die Verhaltensvorschläge sowie Infos zum Ermittlungszeitraum als auch zum Infektionsrisiko finden Nutzer*innen in der App selbst, wenn sie auf „Niedriges **Risik**“ mit dem Finger tippen.

Risik => Risiko